cmd/xenia/scrquery/comments_by_asset.json

```diff
@@ -6,10 +6,10 @@
       {
          "name":"comments_by_asset",
          "type":"pipeline",
-         "collection":"nyt_coral_assets",
+         "collection":"comments",
```

Comment:
Hmm...not sure I follow. Prior to this change, this didn't work because as far as I know, there's no `nyt_coral_assets` collection in more recently-imported data.

...not sure if this is clarifying, but I know @MixedMessages had found that there are some other queries expected from the front-end that don't seem to be in this repo, but this PR is just about fixing the ones that are here.